dojo/tools/acunetix360/parser.py

```diff
@@ -38,7 +38,8 @@ def get_findings(self, filename, test):
             if sev not in ['Info', 'Low', 'Medium', 'High', 'Critical']:
                 sev = 'Info'
             mitigation = text_maker.handle(item.get("RemedialProcedure", ""))
-            references = text_maker.handle(item.get("RemedyReferences", ""))
+            references = "https://online.acunetix360.com/issues/detail/" + item["LookupId"] + "\n"
+            references += text_maker.handle(item.get("RemedyReferences", ""))
```

Comment:
I am not super familiar with acunetix, but is it possible that the lookupID is not present in the report? Maybe something like this would be a tad safer
```suggestion
            references = text_maker.handle(item.get("RemedyReferences", "")) 
            if "LookupId" in item:
                references = f"https://online.acunetix360.com/issues/detail/{item["LookupId"]}\n" + references
```